.circleci/python-only.dockerfile

```diff
@@ -11,6 +11,7 @@ RUN export DEBIAN_FRONTEND=noninteractive && \
         gzip \
         language-pack-en \
         pylint3 \
+        python3-mock \
```

Comment:
IIRC `unittest.mock` is in the standard library in Python 3.

Reply:
Fixed in 4442d3e. Thanks!